src/decoder/plugins/GmeDecoderPlugin.cxx

```diff
@@ -105,16 +105,43 @@ ParseContainerPath(Path path_fs)
 	return { path_fs.GetDirectoryName(), track - 1 };
 }
 
-static void
-gme_file_decode(DecoderClient &client, Path path_fs)
-{
-	const auto container = ParseContainerPath(path_fs);
+static Music_Emu
```

Comment:
Coding style violation: the `*` belongs after the `Music_Emu`. This is unreadable!